pynsee/utils/_wait_api_query_limit.py

```diff
@@ -11,15 +11,19 @@
 from pynsee.utils._create_insee_folder import _create_insee_folder
 from pynsee.utils._hash import _hash
 
+import logging
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def _warning_query_limit():
-    print("\nAPI query number limit reached - function might be slowed down")
+    logger.debug(

```

Comment:
this must be warning

Reply:
At least info, sure; but it will interfere with all tqdm's progressbar and this is in accordance with API's characteristics, so I'm not sure it needs to be warning.

Reply:
mhh... tqdm is notoriously bad with logging... maybe we should move to [alive-progress](https://github.com/rsalmei/alive-progress/) but that's a discussion for another time.

I'd still think this makes things slower than what the person using the tlibrary would expect, so making the message visible is better but I don't have a very strong opinion about it...

cc @hadrilec